canopy_dispatch.ml

```diff
@@ -28,7 +28,8 @@ module Make (S: Cohttp_lwt.Server) (C: V1_LWT.CONSOLE) (Disk: V1_LWT.KV_RO)
     let respond_html ~headers ~status ~content ~title =
       store.subkeys [] >>= fun keys ->
       let body = Canopy_templates.main ~config ~content ~title ~keys in
-      S.respond_string ~headers ~status ~body ()
+      let headers = Cohttp.Header.init_with "Content-Type" "text/html; charset=UTF-8" in
```

Comment:
Here, we'd now (since there's already a `headers` flowing in) use `Cohttp.Header.add headers` instead... (same below)